aioesphomeapi/model.py

```diff
@@ -207,26 +207,46 @@ class FanState(EntityState):
 
 
 # ==================== LIGHT ====================
+class LightColorMode(APIIntEnum):
+    UNKNOWN = 0
+    WHITE = 1
+    COLOR_TEMPERATURE = 3
+    COLD_WARM_WHITE = 4
+    RGB = 8
+    RGB_WHITE = 9
+    RGB_COLOR_TEMPERATURE = 11
+    RGB_COLD_WARM_WHITE = 12
+
+
 @dataclass(frozen=True)
 class LightInfo(EntityInfo):
     supports_brightness: bool = False
-    supports_rgb: bool = False
-    supports_white_value: bool = False
-    supports_color_temperature: bool = False
+    supported_color_modes: List[LightColorMode] = converter_field(
+        default_factory=list, converter=list
```

Comment:
```suggestion
        default_factory=list, converter=LightColorMode.convert_list
```

To convert the values from the protobuf enums to our python enums